### DISCUSSION TITLE
Is there a way  to display multiple videos with fullscreen enabled?

### DISCUSSION THREAD

**Discussion by peabeesheetal** [PierfrancescoSoffritti/android-youtube-player#1138] [General]
I am a newbie to android app development.
I need to display multiple videos one beneath the other and also enable fullscreen on all of them (just like youtube)
Is there an example on how to do it?

I am able to play multiple videos but couldn't get it to work with fullscreen enabled for all.

**Comment by PierfrancescoSoffritti** [comment]
Android allows only one full screen video at a time.
